_health-care/_js/components/financial-assessment/ChildInformationSection.jsx

```diff
@@ -1,21 +1,40 @@
 import React from 'react';
 
+import Child from './Child';
 import ErrorableCheckbox from '../form-elements/ErrorableCheckbox';
-import ErrorableSelect from '../form-elements/ErrorableSelect';
-import ErrorableTextInput from '../form-elements/ErrorableTextInput';
-import DateInput from '../form-elements/DateInput';
-import FullName from '../questions/FullName';
-import SocialSecurityNumber from '../questions/SocialSecurityNumber';
-
-import { childRelationships } from '../../utils/options-for-select.js';
-import { isBlank, isValidMonetaryValue } from '../../utils/validations';
-
-// TODO: create unique nodes for each child in applicationData
+import GrowableTable from '../form-elements/GrowableTable.jsx';
 
 class ChildInformationSection extends React.Component {
-  render() {
-    const message = 'Please enter only numbers and a decimal point if necessary (no commas or currency signs)';
+  // TODO(awong): Pull this out into a model.
+  createBlankChild() {
```

Comment:
Nope.

Binding of `this` into the function with that pattern only is necessary if the function itself accesses `this` (this is Javascript being a crappy language such that class methods by default don't get access to `this`...wtf.)

In the case of these functions, they're 100% functional (depend on nothing other than what's passed in) thus they don't need to be bound to `this`.